### PR TITLE
[Yaml] improve changelog and upgrade entries

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

Improves the wording as suggested by @javiereguiluz in https://github.com/symfony/symfony/pull/19538#issuecomment-237769389.

### CODE REVIEW

File: UPGRADE-3.2.md [hunk 1]
```diff
@@ -74,8 +74,9 @@ Validator
 Yaml
 ----
 
- * Support for silently ignoring duplicate keys in YAML has been deprecated and
-   will lead to a `ParseException` in Symfony 4.0.
+ * Mappings with a colon (`:`) that is not followed by a whitespace are deprecated
```
Comment (Nicofuma) [inline UPGRADE-3.2.md]: I don't think you are talking about the same thing than before

Comment (Nicofuma) [inline UPGRADE-3.2.md]: actually you just removed the wrong entry (see the following one)

File: UPGRADE-3.2.md [hunk 2]
```diff
@@ -74,8 +74,9 @@ Validator
 Yaml
 ----
 
- * Support for silently ignoring duplicate keys in YAML has been deprecated and
```
Comment (Ener-Getick) [inline UPGRADE-3.2.md]: BTW can't we add an example for this one? It looks hard to understand when you're not aware of the linked change.

File: UPGRADE-3.2.md (hunk 1 above)
Comment (xabbuh) [inline UPGRADE-3.2.md]: Oh indeed, good catch. Thank you for the heads up.

**Comment by HeahDude** [comment]
👍

Status: reviewed

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**